bugbug/bugzilla.py

```diff
@@ -5,12 +5,12 @@
 
 import json
 import os
-
+import time
 import requests
 from libmozdata import bugzilla
 
 from bugbug import db
-
+from tqdm import tqdm
 BUGS_DB = 'data/bugs.json'
```

Comment:
There's probably going to be a flake8 error here, as you removed the newline after the imports.